parser/sql/dialect/mysql/src/test/java/org/apache/shardingsphere/sql/parser/mysql/visitor/format/MySQLFormatVisitorIT.java

```diff
@@ -76,23 +76,23 @@ public Stream<? extends Arguments> provideArguments(final ExtensionContext exten
                                     + "`runoob_test` NATIONAL CHAR(40),\n"
                                     + "`submission_date` DATE,\n"
                                     + "PRIMARY KEY (`runoob_id`)\n"
-                                    + ")ENGINE=InnoDB DEFAULT CHARSET=utf8;",
+                                    + ")ENGINE=INNODB DEFAULT CHARSET=utf8;",
```

Comment:
Why modify InnoDB to INNODB?

Reply:
When modifying the vocabulary, this test did not pass, and now the previous modifications have been restored :blush: